core/templates/components/lockup/lockup.twig

```diff
@@ -0,0 +1,65 @@
+{#
+/**
+ * @file
+ * Lockup Component.
+ *
+ * A lockup with the Stanford wordmark logo and department/site name. It links to the homepage of the department/unit.
+ *
+ * Available variants for the lockup:
+ * - .su-lockup - Lockup with one line to the right of the wordmark.
+ * - .su-lockup--two-lines - Lockup with two lines to the right of the wordmark.
+ *
+  * Available variants for the individual site name lines:
+ * - .su-lockup__sitename-line1 - Regular font style for line 1
+ * - .su-lockup__sitename-line1-small - Small font size for line 1
+ * - .su-lockup__sitename-line1-supersmall - Supersmall font size for line 1
+ * - .su-lockup__sitename-line1-uppercase - Uppercase and bold font for line 1
+ * - .su-lockup__sitename-line2 - Regular font style for line 2
+ * - .su-lockup__sitename-line5 - Regular font style for line 5
+ * - .su-lockup__sitename-line5-uppercase - Uppercase, bold and smaller font for line 5
+ *
+ * Available variables:
+ * - link: The URL that the lockup links to.
```

Comment:
```suggestion
 * - attributes: For additional HTML attributes not already provided.
 * - modifier_class: Additional css classes to change look and behaviour.
 * - .su-lockup__sitename-line1 - Regular font style for line 1
 * - .su-lockup__sitename-line1-small - Small font size for line 1
 * - .su-lockup__sitename-line1-supersmall - Supersmall font size for line 1
 * - .su-lockup__sitename-line1-uppercase - Uppercase and bold font for line 1
 * - .su-lockup__sitename-line2 - Regular font style for line 2
 * - .su-lockup__sitename-line5 - Regular font style for line 5
 * - .su-lockup__sitename-line5-uppercase - Uppercase, bold and smaller font for line 5
```

Reply:
Add attributes and modifier_class variables.

Reply:
Thanks for catching that!